src/PsTool/Common.cs

```diff
@@ -5143,20 +5145,23 @@ public static void LoadHyphenationSettings()
 				try
 				{
 					IDictionary<string, string> value = new Dictionary<string, string>();
-					var flexScan = new FlexScan(CallerSetting.SettingsFullPath);
-					foreach (var lang in flexScan.VernWs)
-					{
-						var langname = GetLanguageValues(lang);
-						if (!string.IsNullOrEmpty(lang) && (langname != null) && !value.ContainsKey(lang))
-							value.Add(lang, langname.Replace(',', ' '));
-					}
-					foreach (var lang in flexScan.AnalWs)
+					if (File.Exists(CallerSetting.SettingsFullPath))

```

Comment:
1. The file should always exists if the type of data is DataCreator.CreatorProgram.Fieldworks since it is the FWDATA file.

2. The try command at line 5145 should catch this error, if it ever did occur.

I am not sure what test case will require this line?